app/Http/Controllers/RegisterController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Validation\Rules\Password;
+
+use App\Models\User;
+
+class RegisterController extends Controller
+{
+	public function register(Request $request)
+	{
+		$credentials = $request->validate([
+			"email" => [
+				"required",
+				"email",
+				"regex:/[a-z0-9]+@(ogt013|roctilburg|onderwijsgroeptilburg)\.nl/i",
+			],
+			"password" => [
+				"required",
+				Password::min(8)
+					->letters()
+					->numbers()
+					->uncompromised(),
+				"confirmed",
+			],
+		]);
+
+		if (User::where("email", $credentials["email"])->first() !== null) {
```

Comment:
Instead of if statement, use laravel validation rule `unique`: https://laravel.com/docs/9.x/validation#rule-unique. Like this: `unique:users,email` or `"unique:" . User::class . ",email"`